release/downloadIstioCtl.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+#
+# Separate downloader for istioctl
+#
+# This file will be fetched as: curl -L https://git.io/getLatestIstioCtl | sh -
+#
+
+# Determines the operating system.
+OS="$(uname)"
+if [ "x${OS}" = "xDarwin" ] ; then
```

Comment:
why `x`?

Reply:
actually just borrowed from `downloadIstioCandidate.sh`.  since we have `"`, seems we can remove `x`. but `x` is no harm here I think.